etcdctl/command/util.go

```diff
@@ -65,7 +65,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "127.0.0.1:4001,127.0.0.1:2379"
```

Comment:
this is for backward compatibility. etcdctl should be able to do requests to old etcd cluster.